src/rules/prefer_logical_operator.coffee

```diff
@@ -0,0 +1,37 @@
+module.exports = class PreferLogicalOperator
+
+    rule:
+        name: 'prefer_logical_operator'
+        level: 'ignore'
+        message: 'Don\'t use is, isnt, not, and, or, yes, on, no, off'
+        doubleNotLevel: 'ignore'
+        description: '''
+            This rule prohibits is, isnt, not, and, or, yes, on, no, off.
+            Use ==, !=, !, &&, ||, true, false instead.
+            '''
+
+    tokens: ['COMPARE', 'UNARY', 'LOGIC', 'BOOL', 'COMPOUND_ASSIGN']
+
+    lintToken: (token, tokenApi) ->
+        # console.log "lintToken :: ", token
+        config = tokenApi.config[@rule.name]
+        level = config.level
```

Comment:
I don't think you need to do this, the linter already handles the error level if there is an error thrown.